TimberAPI/CustomObjectSystem/CustomObjectCollection.cs

```diff
@@ -11,7 +10,9 @@ public class CustomObjectCollection : IObjectCollection
 
 		public IEnumerable<Object> GetObjects()
 		{
-			return TimberAPI.CustomObjectRegistry.GetAllGameObjects();
+			List<GameObject> objects = TimberAPI.CustomObjectRegistry.GetAllGameObjects();
```

Comment:
I can't state how much I dislike this. Still think we could just remove this and people could just use `TimberAPI.CustomObjectRegistry.GetAllGameObjects()`

Reply:
This is the IObjectCollection interface - this is how I give the objects to the game

Reply:
This isn't accessible via API